luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/CountMakerFactory.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.data.config.metric.makers.CountMaker;
+import com.yahoo.bard.webservice.data.config.metric.makers.MetricMaker;
+
+/**
+ * Factory that supports LongSumMetricMaker.
```

Comment:
Should refer to the CountMaker, not LongSum.

Reply:
done.